lang/java/reef-examples/src/main/java/org/apache/reef/examples/group/bgd/BGDClient.java

```diff
@@ -67,7 +67,6 @@ public BGDClient(@Parameter(InputDir.class) final String input,
    *
    * @param runtimeConfiguration the runtime to run on.
    * @param jobName              the name of the job on the runtime.
-   * @return
```

Comment:
There are more empty return annotations - will you be adding descriptions to them as well?